groups/sig-contributor-experience/groups.yaml

```diff
@@ -69,6 +69,26 @@ groups:
       - pal.nabarun95@gmail.com
       - priyankasaggu11929@gmail.com
 
+  - email-id: kubernetes-github-managment-alerts@kubernetes.io
+    name: kubernetes-github-managment-alerts
+    description: |-
+      Open mailing list for alerts from Kubernetes GitHub management applications. All other posts will be moderated. Please do not attempt to reply to this list.
+
+      For more information, refer https://git.k8s.io/community/github-management
+    owners:
+      - cblecker@gmail.com
+    managers:
+      - killen.bob@gmail.com
+      - madhav.jiv@gmail.com
+      - nikitaraghunath@gmail.com
+      - pal.nabarun95@gmail.com
+      - priyankasaggu11929@gmail.com
+    settings:
```

Comment:
@cblecker @MadhavJivrajani, what moderation level covers this - `Open mailing list for alerts from Kubernetes GitHub management applications. All other posts will be moderated.`

`MessageModerationLevel: "MODERATE_NON_MEMBERS"` ?
Looked for `"MODERATE_ALL_MESSAGES"`, but that might restrict the alert emails too?

Reply:
/hold

Reply:
I made a suggestion based on the dev@ list. We can set `MessageModerationLevel` to `MODERATE_ALL_MESSAGES`, and then once we get the first message, we can approve with a "always allow from this sender". This should prevent spam.

Reply:
added `MessageModerationLevel: "MODERATE_ALL_MESSAGES"` to the settings.